src/validateFilenames.ts

```diff
@@ -52,5 +52,6 @@ function main() {
       )}\n`,
     );
     console.log(prettyPrintFilenameValidationData(filenamesValidationData));
+    process.exit(1);
```

Comment:
@wholesomedev Here is where the script should exit with an error. Otherwise success is assumed.